charts/helm_lib/templates/_module_security_context.tpl

```diff
@@ -48,6 +48,16 @@ securityContext:
   allowPrivilegeEscalation: false
 {{- end }}
 
+{{- /* Usage: {{ include "helm_lib_module_container_security_context_not_allow_privilege_escalation_with_selinux" . }} */ -}}
+{{- /* returns SecurityContext parameters for Container with allowPrivilegeEscalation false and options for SELinux compatibility*/ -}}
+{{- define "helm_lib_module_container_security_context_not_allow_privilege_escalation_with_selinux" -}}
```

Comment:
Please add after define
`{{- /* Template context with .Values, .Chart, etc */ -}}`

As in below

Reply:
Also please add test for define
https://github.com/deckhouse/lib-helm/blob/main/tests/README.md

Reply:
And you should rebuild documentation

https://github.com/deckhouse/lib-helm#rebuild-documentation